rero_ils/modules/items/api/circulation.py

```diff
@@ -56,6 +56,10 @@ def wrapper(item, *args, **kwargs):
         """Executed before loan action."""
         # TODO: this code will be enhanced while adding the other actions.
         from . import ItemsSearch
+        if function.__name__ == 'validate_request':
+            # checks if the given loan pid can be validated
+            item.is_action_validate_request_possible(**kwargs)
+
         # the smart checkin requires extra checks and actions before a checkin
         if function.__name__ == 'checkin':
             loan, kwargs = item.prior_checkin_actions(item, **kwargs)
```

Comment:
Why not using `elif`? I think there is more optimization in code from Python. And we cannot be in `validate_request` and `checkin` at the same time. So `elif` is acceptable.